generatorv2/lib/generator.rb

```diff
@@ -38,6 +38,10 @@ def camel_case(str)
     str.split(/[-_]/).map(&:capitalize).join
   end
 
+  def to_snake(str)
+    str.gsub(/([a-z\d])([A-Z])/, '\1_\2').tr("-", "_").downcase
```

Comment:
Why not...

```suggestion
    str.tr("-", "_").downcase
```

Are there tests for this file?